### ISSUE TITLE
npm install noble - errors, root does not have permission. 

### ISSUE THREAD

**Issue by Kepete** [Zzm317/homebridge-mi-flower-mate#6]
I tried to install the plugin. Did all the steps and I got stuck in the 3 step. I tried command "npm install noble" and also "sudo npm install noble". Both gave the same output. See attachment. 

<img width="1279" alt="screen shot 2018-01-24 at 15 05 24" src="https://user-images.githubusercontent.com/32489848/35333792-bff7ece4-0118-11e8-9f76-c0ce5284b5aa.png">

How to fix?

**Comment by Kepete** [comment]
I think I got it somehow figured out. No I get only some warnings. Like this: 

<img width="827" alt="screen shot 2018-01-24 at 16 14 07" src="https://user-images.githubusercontent.com/32489848/35339433-4b5c1692-0129-11e8-926e-8e7331ecd2f5.png">

But the Homebridge won't start. I get "Error: The requested platform 'MiFlowerMatePlatform' was not registered by any plugin."

<img width="1274" alt="screen shot 2018-01-24 at 17 10 18" src="https://user-images.githubusercontent.com/32489848/35339527-808d74dc-0129-11e8-9266-3d397f91837d.png">

**Comment by Zzm317** [comment]
make sure you have installed the plugin

**Comment by Kepete** [comment]
I have Installed all the packages mentioned in the README. Do I have to install something else? 
How do I do that ?

**Comment by Zzm317** [comment]
after you install the packages you have to install the plugin...
The packages are requirements not the plugin

**Comment by Kepete** [comment]
Okay, thanks didn't know that. How do I install the plugin? Can't find any instructions.

**Comment by Zzm317** [comment]
Haven’t you installed any homebridge plugins before???
Just use npm to install

**Comment by Zzm317** [comment]
npm install -g homebridge-mi-flower-mate

**Comment by Kepete** [comment]
Thanks for help. I did it and now get new error. 
<img width="1280" alt="screen shot 2018-01-24 at 17 21 05" src="https://user-images.githubusercontent.com/32489848/35340148-0c94e360-012b-11e8-82d5-d4bb363a1e09.png">

**Comment by Kepete** [comment]
I fixed the issue by upgrading the node version. 

Now I run into the `Unable to load BLE` error. What might be the cause ? 

<img width="856" alt="screen shot 2018-01-24 at 20 18 05" src="https://user-images.githubusercontent.com/32489848/35349561-f4687d92-0143-11e8-94bc-b23f62061e0c.png">

**Comment by Zzm317** [comment]
Unable to load ble means your noble hasn’t be installed correctly

**Comment by Kepete** [comment]
Thanks for your help, I got it working now. 
The main problem was really noble. I got it installed but it was still not working. Later discovered that the only command that installs noble correctly on Raspberry Pi 3 was `sudo npm install -g noble --unsafe-perm` . Nothing other than exactly that. 

Everything is now working but still I don't understand how often it updates the information from the sensor?
